MediaBrowser.Providers/Plugins/AudioDb/ExternalIds.cs

```diff
@@ -62,7 +61,6 @@ public class AudioDbOtherArtistExternalId : IExternalId
         public string UrlFormatString => "https://www.theaudiodb.com/artist/{0}";
 
         /// <inheritdoc />
-        public bool Supports(IHasProviderIds item)
-            => item is Audio || item is MusicAlbum;
+        public bool Supports(IHasProviderIds item) => item is Audio || item is MusicAlbum;
```

Comment:
Why do you make sone one-line funcs in two line, and this one back in one-line? Seems inconsistent.

Reply:
I did this because the pull request was made over the course of three different sessions, and we have no set convention for this syntax.